samples/subsys/event_manager/src/events/ack_event.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _ACK_EVENT_H_
+#define _ACK_EVENT_H_
+
+/**
+ * @brief ACK Event
+ * @defgroup ack_event ACK Event
+ * @{
+ */
+
+#include "event_manager/event_manager.h"
```

Comment:
let's use headers correctly: anything in the include path (non-local) must use `<>`.